sdk/identity/azure-identity-broker/CHANGELOG.md

```diff
@@ -1,15 +1,12 @@
 # Release History
 
-## 1.2.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.1.7 (2024-11-04)
```

Comment:
Update the version in README as well.

Reply:
will be updated post release, as it is wired to pick up released dependency version.